src/Breadcrumbs.tsx

```diff
@@ -31,39 +31,39 @@ const Wrapper = styled.li`
   }
 `
 
-const BreadcrumbBase = styled.nav<SystemFlexProps & SystemCommonProps & SxProp>`
+const BreadcrumbsBase = styled.nav<SystemFlexProps & SystemCommonProps & SxProp>`
   display: flex;
   justify-content: space-between;
   ${COMMON};
   ${FLEX};
   ${sx};
 `
 
-export type BreadcrumbProps = ComponentProps<typeof BreadcrumbBase>
+export type BreadcrumbsProps = ComponentProps<typeof BreadcrumbsBase>
 
-function Breadcrumb({className, children, theme, ...rest}: React.PropsWithChildren<BreadcrumbProps>) {
-  const classes = classnames(className, 'Breadcrumb')
+function Breadcrumbs({className, children, theme, ...rest}: React.PropsWithChildren<BreadcrumbsProps>) {
+  const classes = classnames(className, 'Breadcrumbs')
```

Comment:
I wonder why we need to add this classname 🤔

Reply:
If there are no references it anywhere else in the codebase, let's go ahead and remove it

Reply:
There's a specific test case that validates a ``Breadcrum`` class exists, so it was done on purpose for what it seems. There're other components with readable classes injected to the components. But I don't think it's a convention as other components do not add them.

The removal or rename to ``Breadcrums`` will be breaking if someone relies on the class name for their tests (although it's not a good practice).

Reply:
Let's go ahead and remove it (and the test case) and add a changeset to indicate that it's a breaking change